src/components/shareable-url-modal/shareable-url-modal.js

```diff
@@ -82,6 +72,43 @@ const ShareableUrlModal = ({ onToggleModal, visible }) => {
     fetchPackageCompatibility();
   }, []);
 
+  const displayPublishedView = () => {
```

Comment:
```suggestion
  const setStateForPublishedView = () => {
```

Reply:
I realized we are not displaying any UI but rather setting/updating state. It would be better to rename this to convey the same